src/Controller/CRUDController.php

```diff
@@ -321,12 +320,6 @@ public function editAction($deprecatedId = null) // NEXT_MAJOR: Remove the unuse
 
         $form = $this->admin->getForm();
 
-        if (!\is_array($fields = $form->all()) || 0 === \count($fields)) {
```

Comment:
What is the value returned by `$form->all()` when there are no fields?

Reply:
The signature in symfony is
```
   /**
     * Returns all children in this group.
     *
     * @return self[]
     */
    public function all();
```

So I assume it's always an array ; And in this case it's an empty array.

Reply:
Great. Weird check then. I thought in some circumstances it could be an iterator.